packages/avatar-group/test/avatar-group.test.js

```diff
@@ -273,16 +275,9 @@ describe('avatar-group', () => {
       overflow = group.$.overflow;
     });
 
-    afterEach(() => {
-      overlay.close();
-    });
-
-    it('should open overlay on overflow avatar click', (done) => {
```

Comment:
There is no need to wait for event here, as the `opened` property is updated synchronously.